vm/src/org/mozartoz/truffle/nodes/local/BindOnStackForcedNode.java

```diff
@@ -0,0 +1,30 @@
+package org.mozartoz.truffle.nodes.local;
+
+import org.mozartoz.truffle.nodes.DerefIfBoundNode;
+import org.mozartoz.truffle.nodes.OzNode;
+
+import com.oracle.truffle.api.frame.FrameSlot;
+import com.oracle.truffle.api.frame.VirtualFrame;
+
+public class BindOnStackForcedNode extends OzNode {
```

Comment:
This looks just like InitializeTmpNode but with an additional deref-if-bound (which indeed should be added there).
Should we merge them then?